lib/elixir_boilerplate_graphql/router.ex

```diff
@@ -5,13 +5,6 @@ defmodule ElixirBoilerplateGraphQL.Router do
     @moduledoc false
     use Plug.Router
 
-    plug(
-      Plug.Parsers,
-      parsers: [:json],
-      pass: [],
-      json_decoder: Phoenix.json_library()
-    )
-
```

Comment:
@MathieuLegault1 But we could add a custom plug here to _make sure_ the request content-type is `application/json` right?

Reply:
Yes we could. I'll merge this PR so nobody has this problem if someone forks it and I'll look to do it this week.